Dockerfile

```diff
@@ -4,24 +4,21 @@ ENV GOPATH /go
 
 RUN \
   adduser -h /site -s /sbin/nologin -u 1000 -D hugo && \
-  apk add --no-cache dumb-init && \
-  apk add --no-cache --virtual .build-deps \
-    git \
-    make && \
-  go get github.com/kardianos/govendor && \
-  govendor get github.com/gohugoio/hugo && \
+  apk add --no-cache \
+    dumb-init \
+    git && \
+  go get github.com/gohugoio/hugo && \
```

Comment:
You cannot just "go get" Hugo due to the vendoring. It will never be correct. So while the Docker file may be smaller, it is now not correct.